lib/wopinator/lock_id.rb

```diff
@@ -0,0 +1,49 @@
+module Wopinator
+  class LockId
+    def initialize(id)
+      self.id = id.to_s
+    end
+
+    def valid?
+      valid_length? && valid_format? 
+    end
+
+    def present?
+      !empty?
+    end
+
+    def empty?
+      id.bytesize == 0
+    end
+
+    def eql?(other)
+      to_s.eql?(other.to_s)
+    end
+
+    def ==(other)
+      to_s == other.to_s
+    end
+
+    def !=(other)
```

Comment:
Do you need to define both `==` and `!=` if they're truly opposites?  Won't defining `==` be enough?